### PR TITLE
Moving to v0.0.10 - changing download page and associated files.

### PR DESCRIPTION
**Pull request by icarusz**
A few simple changes to the download page and all the associated binaries and checksums.

### CODE REVIEW

**Review by zonotope — APPROVED** [review]
:apple:

File: download.md [hunk 1]
```diff
@@ -19,15 +20,15 @@ You can find basic instructions for
...
+[Tupelo 0.0.10 - Linux - Checksum](tutorials/download/tupelo-v0.0.10-linux-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
 
 ***
 
 ## OSX
 
-[Tupelo 0.0.9 - macOS](tutorials/download/tupelo-v0.0.9-darwin-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
-[Tupelo 0.0.9 - macOS - Checksum](tutorials/download/tupelo-v0.0.9-darwin-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
+[Tupelo 0.0.10 - OSX](tutorials/download/tupelo-v0.0.10-darwin-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
+[Tupelo 0.0.10 - OSX - Checksum](tutorials/download/tupelo-v0.0.10-darwin-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
```
Comment (cap10morgan) [inline download.md:32]: Ack! Naming regression! :)

File: download.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ nav_order: 4
 These binaries are all you need to get started signing chain-trees and running
 your own local test net.
 
-The most recent release (0.0.9) includes several bug fixes.
+The most recent release (0.0.10) includes several bug fixes related to changing
+ownership and importing/exporting chain-trees.
```
Comment (cap10morgan) [inline download.md:14]: Not a big deal but the "changing ownership" bug ended up being much broader than that. Any addition of new top-level keys (the first time you change owners or establish a coin does this implicitly) used to drop any existing ones.

File: download.md (hunk 1 above)
Comment (icarusz) [reply download.md:32]: DOH! Fixorated.

**Pull request merged**